cache-core/src/main/java/io/micronaut/cache/CacheConfiguration.java

```diff
@@ -106,6 +107,13 @@ public Optional<Duration> getExpireAfterWrite() {
         return Optional.ofNullable(expireAfterWrite);
     }
 
+    /**
+     * @return The duration after an entry is eligible for refresh
+     */
+    public Optional<Duration> getRefreshAfterWrite() {
```

Comment:
See caffeine specific, can the new getter/setter go in the caffeine specific subclass?